tests/README.md

```diff
@@ -0,0 +1,55 @@
+# Ansible FreeIPA Test Playbooks
+
+This directory contains a series of playbooks used to test the idempotency of roles provided by ansible-freeipa.
+
+
+## Requirements
+
+In order to run those tests you will need to have `vagrant` installed with the plugins `vagrant-hostmanager` and `vagrant-libvirt`.
```

Comment:
Don't know how to rephrase it, but `vagrant` is not needed for the tests, it is needed for the automation being created.